comment_scan/replier.rb

```diff
@@ -0,0 +1,296 @@
+require './db'
+require 'httparty'
+require 'htmlentities'
+
+class Replier
```

Comment:
This looks awfully similar to commander. Could we build some common abstraction around that?

Reply:
Yeaaaaa I was thinking about this too. I'm not sure how I'd do it though. There are enough quirks in each that I'm not sure if it'd be worth it to try to create something more generic just for the two of them (especially considering we should never have another class like this).